[
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/791484249",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/6910#issuecomment-791484249",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/6910",
        "id": 791484249,
        "node_id": "MDEyOklzc3VlQ29tbWVudDc5MTQ4NDI0OQ==",
        "user": {
            "login": "hariharans29",
            "id": 9969784,
            "node_id": "MDQ6VXNlcjk5Njk3ODQ=",
            "avatar_url": "https://avatars.githubusercontent.com/u/9969784?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/hariharans29",
            "html_url": "https://github.com/hariharans29",
            "followers_url": "https://api.github.com/users/hariharans29/followers",
            "following_url": "https://api.github.com/users/hariharans29/following{/other_user}",
            "gists_url": "https://api.github.com/users/hariharans29/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/hariharans29/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/hariharans29/subscriptions",
            "organizations_url": "https://api.github.com/users/hariharans29/orgs",
            "repos_url": "https://api.github.com/users/hariharans29/repos",
            "events_url": "https://api.github.com/users/hariharans29/events{/privacy}",
            "received_events_url": "https://api.github.com/users/hariharans29/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2021-03-05T15:17:24Z",
        "updated_at": "2021-03-05T15:17:24Z",
        "author_association": "MEMBER",
        "body": "Not sure how the exported models differ with/without the offending line. This seems like something to bring to the attention of the Torch exporter project perhaps. \r\n\r\n\"\r\nonnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /Users/runner/work/1/s/onnxruntime/core/providers/cpu/tensor/reshape_helper.h:43 onnxruntime::ReshapeHelper::ReshapeHelper(const onnxruntime::TensorShape &, std::vector<int64_t> &) gsl::narrow_cast<int64_t>(input_shape.Size()) == size was false. The input tensor cannot be reshaped to the requested shape. Input shape:{}, requested shape:{25}\"\r\n\r\nAll this line is telling you is that somewhere in the broken model there is a request to reshape a scalar (shape: {}) to a 1-D tensor (shape: {25}) which is of course not possible and hence the complaint from ORT when trying to load the model. ",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/791484249/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    }
]